docker-compose.yml

```diff
@@ -42,17 +42,9 @@ services:
     image: postgres:14.4-alpine3.16
     restart: always
     user: postgres
-    volumes:
-      - transcendence-db:/var/lib/postgresql/data
-      - ./db/fill-db.sh:/docker-entrypoint-initdb.d/fill-db.sh
-      - ./db/setup/:/var/lib/postgresql/setup/
     environment:
       - POSTGRES_DB=ft_transcendence
 
   memcached:
     image: memcached:1.6.15-alpine3.16
     restart: always
-
-volumes:
-  transcendence-db:
-  transcendence-app-data:
```

Comment:
Why did you differentiate between dev and prod volumes?

Reply:
Knex saves the migration filenames and executes them once. Unfortunately, it also saves the extension. In development environment, we use typescript extensions, and in production javascript.

If we want to test both dev and prod environment on the same machine, it throws the following error:

```
Error: The migration directory is corrupt, the following files are missing: migration.[ts, js]*
```

* `.ts` or `.js` depending on which environment we run first